### PR TITLE
[OSDOCS-7140]: Move AWS tasks for hosted control planes to AWS subsection

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-7140

Link to docs preview: https://62943--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-configuring.html

QE review:
- [x] QE has approved this change.

Additional information: This PR moves 3 modules that are AWS-specific to the AWS subsection of the "Configuring hosted control planes" book. It also removes some of the Technology Preview wording, as hosted control planes will be GA for 4.14.

### CODE REVIEW

**Comment by jiezhao16** [comment]
lgtm

**Review by zanetworker — APPROVED** [review]

File: hosted_control_planes/hcp-configuring.adoc [hunk 1]
```diff
@@ -8,29 +8,29 @@ toc::[]
...
-* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#hosted-control-planes-manage-aws[Managing hosted control plane clusters on AWS (Technology Preview)]: Management tasks include creating, importing, accessing, or deleting a hosted cluster on AWS.
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#hosting-service-cluster-configure-aws[Configuring the hosting cluster on AWS]: The tasks to configure a hosted cluster on AWS include creating the AWS S3 OIDC secret, creating a routable public zone, enabling external DNS, enabling AWS PrivateLink, enabling the hosted control planes feature, and installing the hosted control planes CLI.
```
Comment (nalhadef) [inline hosted_control_planes/hcp-configuring.adoc:17]: Should OIDC and DNS get a spelling out for first usage?

File: hosted_control_planes/hcp-configuring.adoc [hunk 2]
```diff
@@ -8,29 +8,29 @@ toc::[]
...
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#hosted-control-planes-manage-aws[Managing hosted control plane clusters on AWS]: Management tasks include creating, importing, accessing, or deleting a hosted cluster on AWS.
+* xref:../networking/hardware_networks/configuring-sriov-operator.adoc#sriov-operator-hosted-control-planes_configuring-sriov-operator[Deploying the SR-IOV Operator for hosted control planes]: After you configure and deploy your hosting service cluster, you can create a subscription to the SR-IOV Operator on a hosted cluster. The SR-IOV pod runs on worker machines rather than the control plane.
```
Comment (nalhadef) [inline hosted_control_planes/hcp-configuring.adoc:19]: Does SR-IOV need to be spelled out?

File: hosted_control_planes/hcp-configuring.adoc [hunk 3]
```diff
@@ -8,29 +8,29 @@ toc::[]
...
-* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#hosted-control-planes-manage-bm[Managing hosted control plane clusters on bare metal (Technology Preview)]: Create a hosted cluster, create an `InfraEnv` resource, add agents, access the hosted cluster, scale the `NodePool` object, handle Ingress, enable node auto-scaling, or delete a hosted cluster.
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-configure-bm[Configuring the hosting cluster on bare metal]: Configure DNS before you create a hosted cluster. 
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#hosted-control-planes-manage-bm[Managing hosted control plane clusters on bare metal]: Create a hosted cluster, create an `InfraEnv` resource, add agents, access the hosted cluster, scale the `NodePool` object, handle Ingress, enable node auto-scaling, or delete a hosted cluster.
```
Comment (nalhadef) [inline hosted_control_planes/hcp-configuring.adoc:28]: Consider changing the serial list to an unordered list.

File: hosted_control_planes/hcp-configuring.adoc (hunk 2 above)
Comment (lahinson) [reply hosted_control_planes/hcp-configuring.adoc]: @nalhadef Good point. I updated the file to spell it out here for users who might not be familiar with SR-IOV.

File: hosted_control_planes/hcp-configuring.adoc (hunk 3 above)
Comment (lahinson) [reply hosted_control_planes/hcp-configuring.adoc]: Done. I also realized that the information in those topics changed slightly, so I updated the descriptions of both the "Configuring the hosting cluster on bare metal" and "Managing hosted control plane clusters on bare metal" topics to include unordered lists.

**Comment by ousleyp** [comment]
LGTM!

**Pull request merged**